src/test/java/io/camunda/zeebe/bpmnassert/util/Utilities.java

```diff
@@ -146,22 +149,26 @@ public static PublishMessageResponse sendMessage(
     return response;
   }
 
-  public static void increaseTime(final ZeebeEngine engine, final Duration duration) {
-    engine.clock().increaseTime(duration);
+  public static void increaseTime(final InMemoryEngine engine, final Duration duration) {
+    engine.increaseTime(duration);
     waitForIdleState(engine);
   }
 
   public static void completeTask(
-      final ZeebeEngine engine, final ZeebeClient client, final String elementId) {
-    Record<JobRecordValue> lastRecord = null;
-    for (final Record<JobRecordValue> record : engine.jobRecords().withElementId(elementId)) {
-      if (record.getIntent().equals(JobIntent.CREATED)) {
-        lastRecord = record;
-      }
-    }
-    if (lastRecord != null) {
+      final InMemoryEngine engine, final ZeebeClient client, final String elementId) {
+    final List<Record<JobRecordValue>> records =
+        StreamFilter.jobRecords(engine.getRecordStream())
+            .withElementId(elementId)
+            .withIntent(JobIntent.CREATED)
+            .stream()
+            .collect(Collectors.toList());
+
+    if (!records.isEmpty()) {
```

Comment:
:x: I think we need some kind of error handling or feedback so that callers of this method know whether it succeeded or not.